src/gda/backend_gda.hpp

```diff
@@ -96,6 +91,13 @@ class GDAHostContext;
 class QueuePair;
 class HostInterface;
 
+enum GDA_NIC {
```

Comment:
```suggestion
enum class GDA_NIC {
```
can maybe bikeshed naming, but don't really know if e.g. `GDAType`, `GDAVendor`, etc. is any better

Reply:
Yup, I should adjust the name to be a little more consistent with the code base